src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/pact/PactContractTest.kt

```diff
@@ -15,8 +15,6 @@ import org.springframework.context.annotation.Import
 import org.springframework.http.HttpHeaders
 import org.springframework.test.context.ActiveProfiles
 import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.common.config.JwtAuthHelper
-import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.domain.repository.CourseRepository
```

Comment:
One of the things we try to do as part of the atomic-commits process is squashing later changes down that aren't directly adding functionality. For instance, you could probably use `git rebase -i main` to make fixup commits and squash down [de4ab75](https://github.com/ministryofjustice/hmpps-accredited-programmes-api/pull/155/commits/de4ab75806abfc9c1383294b2a16aaa52a6f9f8b) into any of the earlier ones. There's an excellent tool out there that can do most of this heavy lifting for you called [`git absorb`](https://github.com/tummychow/git-absorb). Happy to go through it with you at some point if you like.

Reply:
I haven't used git absorb. I am used to using `git rebase`
Happy to pair with you on using git absorb